client/src/featureform/register.py

```diff
@@ -1100,8 +1100,9 @@ def __call__(self, fn: Callable[[], str]):
     def __set_query(self, query: str):
         if query == "":
             raise ValueError("Query cannot be an empty string")
-        if not self._is_valid_sql_query(query):
-            raise InvalidSQLQuery(query)
+        # The regex here is wrong, needs to be fixed
+        # if not self._is_valid_sql_query(query):
```

Comment:
Was there no test for this? If there was, add the case where you found it to be broken into the test.

Reply:
I updated the regex and tests. Please re-review.